pkg/plugin/datasource.go

```diff
@@ -104,7 +101,7 @@ func (ds *MQTTDatasource) CheckHealth(ctx context.Context, req *backend.CheckHea
 	}, nil
 }
 
-func (ds *MQTTDatasource) SubscribeStream(ctx context.Context, req *backend.SubscribeStreamRequest) (*backend.SubscribeStreamResponse, error) {
+func (ds *MQTTDatasource) SubscribeStream(_ context.Context, req *backend.SubscribeStreamRequest) (*backend.SubscribeStreamResponse, error) {
 	ds.Client.Subscribe(req.Path)
 
```

Comment:
I think based on our discussion, we should remove this, since now the RunStream takes care of the topic subscription once.
```suggestion
```

Reply:
Let's do